exercises/APE1/0102.tex

```diff
@@ -0,0 +1,3 @@
+White pawn in C4, black king in row 6, white rook in G6, black king in row 7, white rook in G7. Then if the black king goes in B8, white bishop in D6, Checkmate. Else if the black king goes in row 6, white rook in D6, Checkmate.
+
+In game theory, the opponent is always considered rational and intelligent. In our case, the opponent will be considered a great master of chess, not a novice.
```

Comment:
In this exercise we also consider the case of the son that won't play the knight.
Explain what would happen if you play the solution of exercise one for this chess scenario and what the chess grandmaster could do with his knight.